README.md

```diff
@@ -12,69 +12,13 @@ npm install --save-dev @enact/jsdoc-to-ts
 
 ## Usage
 
-Assuming this directory is a peer of the Enact source and you want to write this into an installed (from npm) version of enact:
+Assuming this directory is a peer of the Sandstone source and you want to write this into a same path of sandstone:
```

Comment:
Why are you changing enact to sandstone?

Reply:
An example of enact is as follows. 
node bin/jsdoc-to-ts.js ../enact/core -o=../enact/core
node bin/jsdoc-to-ts.js ../enact/ui-o=../enact/ui
node bin/jsdoc-to-ts.js ../enact/i18n -o=../enact/i18n 
node bin/jsdoc-to-ts.js ../enact/webos -o=../enact/webos 
node bin/jsdoc-to-ts.js ../enact/spotlight -o=../enact/spotlight 

Sandstone looks good to show the simplicity of the CLI method.